draft-barnes-acme.md

```diff
@@ -1162,7 +1161,7 @@ The client needs to respond with information to complete the challenges.  To do
 this, the client updates the authorization object received from the server by
 filling in any required information in the elements of the "challenges"
 dictionary.  For example, if the client wishes to complete the "simpleHttp"
-challenge, it needs to provide the "path" component.  (This is also the stage
+challenge, it needs to provide the "tls" component.  (This is also the stage
```

Comment:
I think this is not accurate. For better or for worse, the `tls` field is currently specified as optional, defaulting to true.